app/navigation/Navigator.scala

```diff
@@ -363,4 +349,34 @@ class Navigator @Inject() () {
       case None     => controllers.addFinancialInstitution.registeredBusiness.routes.RegisteredBusinessCheckYourAnswersController.onPageLoad()
     }
 
+  private def firstContactHavePhoneCheckModeNavigation(userAnswers: UserAnswers): Call =
```

Comment:
you can probably avoid merging the have phone and phone page objects but it's readable as it - is this just a refactor so the tests pass or are we perhaps missing tests? 😵